### ISSUE TITLE
Differences implement between paper and code 

### ISSUE THREAD

**Issue by AHupuJR** [sebgao/LIP#6]
Hello there, thanks for your great works at first!
I found a difference between paper and code: In the paper, the Projection Logit Module is made of 1 by 1 conv followed by Affined IN and Amplified Sigmoid function; however related codes showed as bellow:
```
class ProjectionLIP(nn.Module):
    def __init__(self, channels):
        super(ProjectionLIP, self).__init__()

        rp = channels

        self.logit = nn.Sequential(
            OrderedDict((
                ('conv', nn.Conv2d(channels, channels, 3, padding=1, bias=False)),
                ('bn', nn.InstanceNorm2d(channels, affine=True)),
                ('gate', SoftGate()),
            ))
        )

    def init_layer(self):
        self.logit[0].weight.data.fill_(0.0)

    def forward(self, x):
        frac = lip2d(x, self.logit(x))
        return frac
```
which is a 3 by 3 conv instead of 1 by 1 conv, why?

**Comment by sebgao** [comment]
Hi, thank you for your interest in our work.

The code only shows the structure of ResNet with **the bottleneck form** of LIP logit modules, that is, LIP-ResNet w Bottleneck-128 here. So the logit module in `ProjectionLIP` is actually a simplified version of the bottleneck form, as pointed out by the paper

> Bottleneck-x logit module in LIP substitution for replacing max pooling in ResNet and DenseNet is simply a 3 × 3 convolution.

(page 5, the end of the paragraph which begins with **Design of logit modules**)

Sorry for the confusion here. We should name `ProjectionLIP` something else.

**Comment by AHupuJR** [comment]
Thank you for reply, I modify that 3 by 3 conv to a 1 by 1 conv as a real ProjectionLIP Module in my attempt to make a resnet18+LIP.

**Comment by sebgao** [comment]
Actually you can abandon the LIP substitution for max pooling at the cost of top-1 accuracy drop and nearly the same top-5 accuracy (shown in Table 3 in the paper), if the computation burden of LIP for max pooling is unbearable .

**Comment by AHupuJR** [comment]
got it, thanks!
